Process.c

```diff
@@ -707,6 +707,21 @@ void Process_printLeftAlignedField(RichString* str, int attr, const char* conten
    RichString_appendChr(str, attr, ' ', width + 1 - columns);
 }
 
+void Process_printPercentage(float val, char* buffer, int n, int* attr) {
+   if (val >= 0) {
+      if (val < 100) {
+         xSnprintf(buffer, n, "%4.1f ", val);
+      } else if (val < 1000) {
+         xSnprintf(buffer, n, "%3d. ", (unsigned int)val);
+      } else {
+         xSnprintf(buffer, n, "%4d ", (unsigned int)val);
```

Comment:
"%d" expects an `int` argument, not `unsigned int`.
```suggestion
         xSnprintf(buffer, n, "%3d. ", (int)val);
      } else {
         xSnprintf(buffer, n, "%4d ", (int)val);
```